_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,183 @@
+    likely an increased reliance on reputation---which likely
+    disproportionately benefits larger nodes.  We're sure developers will
+    continue working on this important issue and we'll report on any
+    notable progress in future newsletters.
+
+- **Simplified HTLC negotiation:** Rusty Russell [posted][russell
+  simplified] to the Lightning-Dev mailing list about simplifying how
+  payments are made and resolved in LN channels.  Currently, both sides
+  of a channel can each propose relaying or settling a payment, which
+  requires creating or removing an [HTLC][topic htlc] from the offchain
+  commitment transaction.  Sometimes both sides propose a change at the
+  same time and so there are conflicting commitment transactions.  These
+  potential conflicts may get more complicated if dynamic changes to the
+  commitment transaction format and settings are allowed (see
+  [Newsletter #108][news108 upgrade commitment]).  One proposal to
+  eliminate that complication is to only allow dynamic changes when all
+  HTLCs have been resolved and the channel is quiet---that improves
+  safety but is more restrictive than desired.
+
+    Russell's proposal this week was to instead only allow one party in
+    a channel to propose HTLC changes.  This eliminates the risk of
+    conflicting proposals but it increases the average network
+    communication overhead by half a round trip (assuming each party
+    proposes an equal number of HTLC changes).  Responsibility for
+    proposing changes can be transferred from one party to the other as
+    needed, allowing each party to propose channel updates at different
+    times.
+
+    As of this writing, the proposal received only a small amount of
+    discussion from other LN implementation maintainers, so its future
+    remains uncertain.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19953][] implements [schnorr signature
+  verification][topic schnorr signatures] ([BIP340][]), [taproot][topic
+  taproot] verification ([BIP341][]), and [tapscript][topic tapscript]
+  verification ([BIP342][]).  The new soft fork rules are currently only
+  used in the private regression test mode (regtest); they are not
```

Comment:
Suggest removing 'regression' here. That's the origin of the word 'regtest', since it was initially used with the bitcoinj comparison tool to find behavior changes (https://github.com/bitcoin/bitcoin/commit/0e4b31755534fac4ea6c20a60f719e3694252220), but we use it for all kinds of testing now.